aim/cli/storage/commands.py

```diff
@@ -55,22 +55,32 @@ def to_3_11(ctx, hashes):
     repo = Repo.from_path(repo_path)
 
     matched_hashes = match_runs(repo_path, hashes)
+    remaining_runs = []
     confirmed = click.confirm(f'This command will optimize the metrics data for {len(matched_hashes)} runs from aim '
                               f'repo located at \'{repo_path}\'. This process might take a while. '
                               f'Do you want to proceed?')
     if not confirmed:
         return
 
     for run_hash in tqdm(matched_hashes):
-        run = MaintenanceRun(run_hash, repo=repo)
-        if run.check_metrics_version():
-            backup_run(run)
-            run.update_metrics()
-        else:
-            click.echo(f'Run {run.hash} is already up to date. Skipping')
-    click.echo('Finished optimizing metric data. '
-               'In case of any issues the following command can be used to restore data:')
-    click.secho(f'aim storage --repo {repo.root_path} restore \'*\'', fg='yellow')
+        try:
+            run = MaintenanceRun(run_hash, repo=repo)
+            if run.check_metrics_version():
+                backup_run(run)
+                run.update_metrics()
+            else:
+                click.echo(f'Run {run.hash} is already up to date. Skipping')
```

Comment:
Do we want output errors for each run though? Cause in the end there's a message anyway informing that some runs were skipped.